apps/omg_db/lib/db.ex

```diff
@@ -122,6 +110,19 @@ defmodule OMG.DB do
   def get_single_value(parameter_name), do: driver().get_single_value(parameter_name)
   def get_single_value(parameter_name, server), do: driver().get_single_value(parameter_name, server)
 
+  @doc """
+  This is generic DB function that can get the specific data of a specific type.
+  If it is a single value data, use get_single_value/1 instead.
+  """
+  def get(type, specific_keys, timeout), do: driver().get(type, specific_keys, timeout)
```

Comment:
do we want to name this `get_multi` instead?

Reply:
👍 yes, be specific!

Reply:
rename the function to `batch_get`